code/game/objects/items/grenades/plastic.dm

```diff
@@ -85,6 +85,8 @@
 /obj/item/grenade/c4/afterattack(atom/movable/bomb_target, mob/user, flag)
 	. = ..()
 	aim_dir = get_dir(user, bomb_target)
+	if(isobserver(bomb_target))
```

Comment:
```suggestion
	if(isdead(bomb_target))
```

Reply:
`#define isdead(A) (istype(A, /mob/dead))`

No
Subtypes of `/mob/dead` include `observer` and `new_player` which is a holder mob for people in the menu.
I don't think there would ever be a situation where you could plant a c4 on a `new_player` during runtime, and I don't think you could ever click on one do to them having INVISIBILITY_ABSTRACT, 
I have no idea why Cobby thinks this is a necessary change but whatever.